<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginFailed</class>
 <widget class="QDialog" name="LoginFailed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginFailed</string>
  </property>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Login Credentials did not match any registered accounts.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
